{"name": "测试输入：{\"grant_type\":\"password\",\"username\":\"17688701450\",\"password\":\"Sutpc@2020\",\"imageCode\":\"9527\"}", "status": "passed", "parameters": [{"name": "inData", "value": "'{\"grant_type\":\"password\",\"username\":\"17688701450\",\"password\":\"Sutpc@2020\",\"imageCode\":\"9527\"}'"}, {"name": "respData", "value": "'{\"error\": \"invalid_grant\",\"error_description\": \"Bad credentials\"}'"}], "start": 1619077452258, "stop": 1619077452446, "uuid": "904be186-c8fa-48e8-acd1-6143d16b8541", "historyId": "53d81ebd6673670d26d3f2fba159c69e", "testCaseId": "eec049e373b40a29ff5c4ae98ab4d8f6", "fullName": "test_login.TestLogin#test_login_s2", "labels": [{"name": "severity", "value": "critical"}, {"name": "feature", "value": "登录模块"}, {"name": "epic", "value": "TP系统"}, {"name": "story", "value": "登录接口"}, {"name": "tag", "value": "异常账户登陆"}, {"name": "tag", "value": "Login(order=1)"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-5R42TFK"}, {"name": "thread", "value": "13208-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_login"}]}